base/applications/cleanmgr/mainpage.c

```diff
@@ -0,0 +1,103 @@
+/*
+ * PROJECT:         ReactOS Disk Cleanup
+ * LICENSE:         GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:         MainPage child dialog function
+ * COPYRIGHT:       Copyright 2020 Arnav Bhatt (arnavbhatt288 at gmail dot com)
+ */
+
+#include "precomp.h"
+
+DIRSIZE DirectorySizes;
+WCHAR DriveLetter[ARR_MAX_SIZE];
+WCHAR RappsDir[MAX_PATH];
+
+INT_PTR CALLBACK MainPageDlgProc(HWND hwnd, UINT message, WPARAM wParam, LPARAM lParam)
+{
+    HWND hList = GetDlgItem(hwnd, IDC_CHOICE_LIST);
```

Comment:
What happens when the dialog is not initialized yet, or when the dialog proc is called for anything but for using the hList?